esp/services/ws_workunits/ws_workunitsQuerySets.cpp

```diff
@@ -2740,3 +2741,68 @@ bool CWsWorkunitsEx::onWUUpdateQueryEntry(IEspContext& context, IEspWUUpdateQuer
     }
     return true;
 }
+
+bool CWsWorkunitsEx::onWUGetNumFileToCopy(IEspContext& context, IEspWUGetNumFileToCopyRequest& req, IEspWUGetNumFileToCopyResponse& resp)
+{
+    try
+    {
+        Owned<IPropertyTree> result;
+        StringBuffer clusterName = req.getClusterName();
```

Comment:
Actually I misinterpreted this.  You were using the RoxieCluster.  So this is OK.  I'd prefer the parameter name in your ESDL be "RoxieCluster" but we're very inconsistent throughout and Cluster is quite common so doesn't really matter.